### DISCUSSION TITLE
Ability to stockpile raw resources

### DISCUSSION THREAD

**Discussion by ipanic** [Keriew/augustus#261] [Ideas]
When I need raw resouces or plant products to trading / build lighthouse/grand temple, I build more mining buildings. 
The problem is that they are trying to carry resources for processing across the entire map, instead of stockpiling.
I propose to add an option "Stockpiling resources", to move resources directly to the nearest suitable warehouse instead of workshop.

![image](https://user-images.githubusercontent.com/33452459/109395576-7e62ae00-7946-11eb-8da3-419740785bdd.png)

**Comment by crudelios** [comment]
Humm. You mean per building, right?

**Reply by ipanic** [reply]
Right

**Comment by crudelios** [comment]
I thought about this and right now I don't see how to implement this. Even the resources are moved to a warehouse, the warehouse itself will move them to a suitable workshop.

There are only three ways around this:
- Overproduce the raw material
- Stockpile the raw materlal
- Turn off the finished good's industries

Overproducing seems to work best. After a while, your workshops will get so much stock of raw materials that it will be delivered to warehouses instead.

**Reply by ipanic** [reply]
Yes, I think about it too. 
Overproducing  working badly, if you have workshops scattered across the map
Stockpiling the raw material may disable workshops and does not allow to use resource for trading / building
Turning the good's industries off may break houses supply or trading.

> Even the resources are moved to a warehouse, the warehouse itself will move them to a suitable workshop.

We may add fourth button to bottom of the warehouse screen: allow to warehouse worker moving resources.

**Comment by ipanic** [comment]
If it looks useful, I may try to create pull request

**Reply by crudelios** [reply]
Go ahead, it seems like a nice micromanagement option 🙂
